content/oci/oci.go

```diff
@@ -67,6 +82,19 @@ type Store struct {
 	indexLock sync.Mutex
 }
 
+// unsafeStore is used to bypass lock restrictions in Delete.
+type unsafeStore struct {
+	*Store
+}
+
+func (s *unsafeStore) Fetch(ctx context.Context, target ocispec.Descriptor) (io.ReadCloser, error) {
+	return s.storage.Fetch(ctx, target)
+}
+
+func (s *unsafeStore) Predecessors(ctx context.Context, node ocispec.Descriptor) ([]ocispec.Descriptor, error) {
+	return s.graph.Predecessors(ctx, node)
+}
```

Comment:
nit: Let's put this block of code to line 469 so that the code in this file is well organised.

Reply:
Basically, the file body structure is like
```go
package blablabla

import (
   // imports
)

// constants

// global vars

// type definitions (except struct)

// definition of struct A

// methods of struct A

// definition of struct B

// methods of struct B

...

// static private methods
```

Reply:
moved this block.